test/components/DateRangePicker_spec.jsx

```diff
@@ -154,17 +154,32 @@ describe('DateRangePicker', () => {
   });
 
   describe('#clearDates', () => {
+    describe('props.reopenPickerOnClearDates', () => {
```

Comment:
So probably the structure of these tests should look something like:
```
describe('#clearDates', () => {
  describe('props.reopenPickerOnClearDates is truthy', () => {
    describe('props.onFocusChange', () => {
      it('is called once', () => {
        ...
      });
      
      it('is called with arg START_DATE', () => {
        ...
      });
  });

  describe('props.reopenPickerOnClearDates is falsey', () => {
    describe('props.onFocusChange is not called', () => {
      ...
    });
  });
});
```

Some of these tests are duplicates right now.